frontend/src/general-style.css

```diff
@@ -789,6 +789,8 @@ header a:hover {
 	object-fit: cover;
 	object-position: center;
 	margin-bottom: 5px;
+}
+
```

Comment:
unrelated to this PR but noticed an unclosed CSS block

Reply:
yeah i think this was my bad, thanks for fixing

Reply:
yeah this was my bad, thanks for fixing!